src/Libraries/RevitNodes/GeometryConversion/RevitToProtoFace.cs

```diff
@@ -19,42 +19,42 @@ public static class RevitToProtoFace
         {
             if (revitFace == null) throw new ArgumentNullException("revitFace");
 
-            var revitEdgeLoops = EdgeLoopPartition.GetAllEdgeLoopsFromRevitFace(revitFace);
-            var partitionedRevitEdgeLoops = EdgeLoopPartition.ByEdgeLoopsAndFace(revitFace, revitEdgeLoops);
+            var revitCurveLoops = CurveLoopPartition.GetAllCurveloopsFromRevitFace(revitFace);
+            var partitionedRevitCurveLoops = CurveLoopPartition.ByCurveLoopsAndFace(revitFace, revitCurveLoops);
```

Comment:
Could we just use `var curveLoops = face.GetEdgesAsCurveLoops();` to get all curve loops of a face and distinguish the inner and outer? Then use them to create face? I think nested lists or nested nested lists is not a good code practice.